inc/hclib-task.h

```diff
@@ -32,32 +32,20 @@ typedef struct _hclib_task_t {
      * and locality flexible asyncAny
      */
     int is_async_any_type;
```

Comment:
I still don't see an answer to my question. Why weren't they opened one at a time as they became actually mergeable diffs?

And we "stalled" because other things naturally arose Nick. I'm getting back to them now, but these aren't at the top of your or my TODO list.